### ISSUE TITLE
Hook WinstonLutz with cones

### ISSUE THREAD

**Issue by AlainSottiaux** [jrkerns/pylinac#294]
Context:
We implement WL-like test with conical cones collimator (4mm) to check target positionning (by ExacTrac, on a TrueBeam).
Method:
Step 1: position 4mm BB with exactrac, check BB centering in 10mm cone (recenter and reaclibrate if needed).
Step 2: check <2mm BB centering between 10mm cone and 4mm cone.
As pylinac failed to detect BB with our images, we develop our own cone+BB detection, and we calculate ourselve the new couch position to best center BB.
BTW, some core module toosl where really helpful (open images, invert, crop, estimate cone position and size with profile, to start our cone finding, ...)

Is ther a simple way to continue to use our own cone//BB detection, but giving the result to pylinac WL module (so we can reuse output features from pylinac. Basically, we find the center of the cone, and the center of the BB. After that, we calculate new couch coordinates (but pylinac do it also).
As far as I understand, this should correspond to overload _find_field_centroid and _find_bb functions.

If you think it's usefull, I can share with you our cone images, or the method we used to find both cone and BB.

**Comment by jrkerns** [comment]
Yes, this would be handled by overloading the methods you mentioned. I think supporting cones is a wide use case and should be supported. If you [upload](https://www.dropbox.com/request/YKRu4AmuPsXu55uQq761) your cone images I'll take a look to see what needs to be tweaked to get it working.
